BUILDING

```diff
@@ -1 +1,13 @@
-To build Ion, use the build.sh or build.py scripts in the ion/ directory.
+1. Install dependencies
+  - gyp
+  - clang
+  - libc++-dev
+  - libc++abi-dev
+
```

Comment:
Clang 3.3 does work, but needs an adjustment to the build scripts so that it doesn't hang on an incorrect warning.

For the gearsdemo problem, we are aware of this and I'm working on a fix. Feel free to open a separate issue.